src/components/app/details/cdDetails/CDDetails.tsx

```diff
@@ -406,6 +406,7 @@ const HistoryLogs: React.FC<{
 
     return (
         <>
+        {console.log(deploymentAppType, triggerDetails.status)}
```

Comment:
remove this console log & any other if used

Reply:
Done